packages/api/src/context.ts

```diff
@@ -178,6 +182,8 @@ export async function contextFromRequest(
 
       images: new DataLoader(ids => dbAdapter.image.getImagesByID(ids)),
 
+      getCommentsForItemByID: new DataLoader(ids => dbAdapter.comment.getCommentsForItemByID(ids)),
```

Comment:
Your getting comments for **one** Item. Use `id` instead of `ids`

Reply:
fixed

Reply:
fixed